### PR TITLE
Rename connection to session in ParamConverter

### PR DESCRIPTION
**Pull request by Nedeas**
There are a typo in ParamConverter.
In options, it's `connection` which is tested but it's `session` which is to use.

### CODE REVIEW

**Comment by sanpii** [comment]
Can you add a non-regression test?

**Comment by Nedeas** [comment]
I work on.

**Comment by Nedeas** [comment]
Test added !

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
-    - "5.4"
```
Comment (sanpii) [inline .travis.yml]: Is it mandatory?

File: tests/app/autoload.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
```
Comment (sanpii) [inline tests/app/autoload.php]: Why adding this autoloader?

File: .travis.yml [hunk 3]
```diff
@@ -32,6 +31,9 @@ before_install:
     - psql -c 'CREATE DATABASE pomm_test' -U postgres -h 127.0.0.1 postgres
     - psql -c 'CREATE TABLE config (name character varying(25) PRIMARY KEY, value character varying(25))' -U postgres -h 127.0.0.1 pomm_test
     - psql -c "INSERT INTO config VALUES ('test', 'value')" -U postgres -h 127.0.0.1 pomm_test
+    - psql -c 'CREATE DATABASE pomm_test_2' -U postgres -h 127.0.0.1 postgres
```
Comment (sanpii) [inline .travis.yml]: You can add a new connection with the same dsn and, de facto, the same database.

File: .travis.yml (hunk 1 above)
Comment (Nedeas) [inline .travis.yml]: In branch dev-master of pomm are supported php >=5.5.9.
In composer.json when I fork project, it's dev-master referenced in composer.json.
Are you change this ?

File: .travis.yml (hunk 3 above)
Comment (Nedeas) [inline .travis.yml]: Nop. If you use the same database and the same object. How to detect if you use the good session ?

File: tests/app/autoload.php (hunk 2 above)
Comment (Nedeas) [inline tests/app/autoload.php]: For register annotation. See http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html.

File: .travis.yml (hunk 3 above)
Comment (chanmix51) [inline .travis.yml]: Imho they would be two different instances.

Comment (Nedeas) [inline .travis.yml]: @chanmix51 : 2 instances of postgres, databases, ... ?

Comment (chanmix51) [inline .travis.yml]: ah no :) two different Session instances.

Comment (Nedeas) [inline .travis.yml]: Ok ! So It's good ?

File: .travis.yml (hunk 1 above)
Comment (sanpii) [inline .travis.yml]: Yes: https://github.com/pomm-project/pomm-bundle/commit/813299629d6e6021831ab7dc440ed0d783daceea.

Comment (Nedeas) [inline .travis.yml]: I have added your commit and php54 in travis.

**Comment by chanmix51** [comment]
awesome :+1:

**Comment by sanpii** [comment]
I try to merge your PR but your branch have a trange history:
2017-04-12-151608_1676x993_scrot

Can you flaten its?

**Comment by Nedeas** [comment]
@sanpii Done

**Comment by sanpii** [comment]
Nice work, thank you.

**Pull request closed without merging**